.github/workflows/lint.yml

```diff
@@ -11,6 +11,10 @@ jobs:
     if: ${{ github.actor != 'OSBotify' || github.event_name == 'workflow_call' }}
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+        with:
+          fetch-depth: 0
+
```

Comment:
@staszekscp Why do we add this step to all of the workflows?

Reply:
It's because I decided to exclude the `actions/checkout` step from the `setupNode` action. That's why I had to add this line to a couple of workflows. It is not strictly connected to the node setup, and for `pull_request_target` we have to set a `ref` to look at the correct branch for next steps (`pull_request_target` runs in the context of the target branch).

Reply:
Thanks for the context, I am just wondering if we should leave a comment in those workflows to explain why this action needs to be used. At the moment I think it might not be clear. Can you add some comment to this? @staszekscp